tsdb/head_test.go

```diff
@@ -60,6 +60,8 @@ func newTestHead(t testing.TB, chunkRange int64, compressWAL bool) (*Head, *wal.
 	opts.ChunkDirRoot = dir
 	opts.EnableExemplarStorage = true
 	opts.MaxExemplars.Store(config.DefaultExemplarsConfig.MaxExemplars)
+	require.Equal(t, defaultIsolationState, opts.IsolationEnabled)
```

Comment:
`newTestHead` is used by other tests to get a test head. We should not check the default options here.